### ISSUE TITLE
Improve discovery/search of sample CRs

### ISSUE THREAD

**Issue by gberche-orange** [kubeshop/testkube#3537]
**Describe the enhancement you'd like to see**

While the HTML layout at https://docs.testkube.io/test-types/executor-postman/#creating-and-running-test using tab is quite friendly to read as it is compact
![image](https://user-images.githubusercontent.com/4748380/228351099-730d2469-6a19-45c3-a9b6-4c8921823111.png)

it has the drawback of preventing keyword search in the page from the browser unless the reader iterates on all tabs
![image](https://user-images.githubusercontent.com/4748380/228351348-feb5d87c-3e20-4c13-9194-cbc6e991821b.png)

Also I found it hard to find the CR examples, especially as the built-in search seems to not match on CR content

![image](https://user-images.githubusercontent.com/4748380/228352464-203a4048-16eb-467e-a5c4-860eba8ced68.png)
![image](https://user-images.githubusercontent.com/4748380/228352522-086ae3f7-f69a-4377-9c27-610de88c925b.png)

Relates to https://github.com/kubeshop/testkube/issues/3444

**Additional context**
Add any other context, CLI input/output, etc. about the enhancement here.

**Comment by vsukhin** [comment]
thank you @gberche-orange we're improving our docs, still we have a lot of work to complete

**Comment by aabedraba** [comment]
Very good catch @gberche-orange, will definitely think of a solution for this :) Appreciate a lot the feedback!

**Comment by TheBrunoLopes** [comment]
@aabedraba have we fixed this ?

**Comment by aabedraba** [comment]
Currently we're using the Docusaurus tabs feature: https://docusaurus.io/docs/markdown-features/tabs

This improves UX but doesn't allow for search to work on CRDs. 

We will need to replicate the CRDs in a specific section of the Docs so users can search it. 

Will be able to implement by next week.

**Comment by aabedraba** [comment]
TL;DR: `swagger-codgen` generation is not allowing us to create CR documentation.

--- 

The reason why CRs search does not work is because Algolia doesn't include code-blocks in its crawl for result quality reason. 

The solution is to generate CR Documentation in Markdown like this: https://fluxcd.io/flux/components/source/api/v1/

I tried to use the following libraries https://github.com/ahmetb/gen-crd-api-reference-docs and https://github.com/elastic/crd-ref-docs but they both expect a CRD package to be defined in the following standard structure: 

``` 
pkg
    └── apis
        └── mygroup
            └── v1
                ├── doc.go
                ├── register.go
                ├── types.go
                └── zz_generated.deepcopy.go 
```

This structure can be seen in many projects: 
- https://github.com/envoyproxy/gateway/tree/main/api/v1alpha1
- https://github.com/knative/serving/tree/main/pkg/apis/serving/v1alpha1

The problem is that we generate our CRDs in a non-standard way because (mainly) we generate the code using `swagger-codegen`: https://github.com/kubeshop/testkube/blob/develop/Makefile#L89-L111
